Documentation/WritingReST/CheatSheet.rst

```diff
@@ -337,6 +337,21 @@ Source::
       $a = 'hello';
 
 
+Placeholders
+------------
+
+Use the *Backus-Naur form* `<placeholder-name>` for placeholders in code and
+URLs.
+
+Examples:
+
+#. :php:`class <Entity>Controller extends ActionController`
+#. `typo3/sysext/core/bin/typo3 impexp:import <file>`
+#. :samp:`https://example.org/typo3/main?token=<token-id>`
+
+For XML and HTML, use the comment tag :html:`<!-- placeholder-name -->`.
```

Comment:
```suggestion
For XML and HTML, use the comment tag :html:`<!-- placeholder-name -->` since `<placeholder-name>` looks like a regular element.
```

Reply:
In the cheat sheet i would list only the shortest version of the rules and move any explanation to the detailed chapter.

Reply:
Fine with me, wherever it makes the most sense.

Reply:
Yes, it is already in the detailed chapter, see file CodeBlocks.rst:

> In the XML and HTML markup languages, which make extensive use of angle
brackets, the comment tag :html:`<!-- placeholder-name -->` is used to insert
placeholders.

Reply:
Then we should add my suggestion there. I always try to add a justification for rules.

Reply:
Ok, would you suggest your change there?

Reply:
Done.